test/comparison-tests/projectReferencesWatch_Composite_WatchApi/expectedOutput-3.8/patch4/output.txt

```diff
@@ -1,12 +1,6 @@
-                  Asset      Size  Chunks             Chunk Names
-              bundle.js  4.38 KiB    main  [emitted]  main
-            ../app.d.ts  11 bytes          [emitted]  
-../tsconfig.tsbuildinfo  51.8 KiB          [emitted]  
+      Asset      Size  Chunks             Chunk Names
+  bundle.js  4.31 KiB    main  [emitted]  main
+../app.d.ts  11 bytes          [emitted]  
 Entrypoint main = bundle.js
-[./app.ts] 186 bytes {main} [built] [1 error]
```

Comment:
How is this correct change.. Is this issue with the test (where it baselines wrong result) or the change?

Reply:
I believe that's an issue with this test. It should (and still does) produce the expected error. We could revert this make it flaky instead?

Reply:
I've manually verified `projectReferencesWatch_Composite_WatchApi` and  `projectReferencesWatch_WatchApi`, both emit the error when running "manually" but not when using the comparison tests.

Reply:
> Is this issue with the test (where it baselines wrong result) or the change?

I think the problem may be the comparison test pack which seems to be demonstrating unhelpful (incorrect) behaviour. The comparison test pack is not as reliable as I'd like (and I'm partly responsible).

This doesn't give me a warm fluffy feeling but I don't believe it represents a bug in ts-loader.